### PR TITLE
used new logging package in all e2e test main functions

### LINKED ISSUE #3753: Use new logging package in all e2e test `main` functions
**Problem**
In https://github.com/knative-extensions/eventing-kafka-broker/pull/3735 we added a new package which is better at collecting logs, but we aren't calling this everywhere. Specifically, it is only being called in the main function in `test/e2e_new`. We should update the other main functions to set up the logging in a similar way

**[Persona:](https://github.com/knative/eventing/blob/master/docs/personas.md)** 
Which persona is this feature for?
Developers

**Exit Criteria**
All e2e tests successfully collect the logs with the new logging package

**Time Estimate (optional):** 
How many developer-days do you think this may take to resolve? 1

**Additional context (optional)**
Add any other context about the feature request here.

### PR DESCRIPTION
**Pull request by NeerajNagure**
Fixes #3753 

## Proposed Changes
Added the use of new logging package in e2e,e2e_source,e2e_sink,e2e_new_channel,e2e_channel main files and also removed the previous log exporter testlib wherever used in these files
:sparkles:  Used new log package

### CODE REVIEW

File: test/e2e_channel/main_test.go [hunk 1]
```diff
@@ -23,27 +23,39 @@ import (
...
+
+	"knative.dev/eventing-kafka-broker/test/pkg/logging"
+	"knative.dev/reconciler-test/pkg/environment"
 )
 
-var channelTestRunner testlib.ComponentsTestRunner
+// global is the singleton instance of GlobalEnvironment. It is used to parse
+// the testing config for the test run. The config will specify the cluster
+// config as well as the parsing level and state flags.
+var global environment.GlobalEnvironment
```
Comment (Cali0707) [inline test/e2e_channel/main_test.go:39]: Same as above

File: test/e2e_sink/main_test.go [hunk 2]
```diff
@@ -22,15 +22,34 @@ package e2e_sink
...
-	testlib "knative.dev/eventing/test/lib"
-	"knative.dev/pkg/system"
+	"knative.dev/eventing-kafka-broker/test/pkg/logging"
+	"knative.dev/reconciler-test/pkg/environment"
 )
 
+// global is the singleton instance of GlobalEnvironment. It is used to parse
+// the testing config for the test run. The config will specify the cluster
+// config as well as the parsing level and state flags.
+var global environment.GlobalEnvironment
```
Comment (Cali0707) [inline test/e2e_sink/main_test.go:37]: Same as above

File: test/e2e_source/main_test.go [hunk 3]
```diff
@@ -22,15 +22,34 @@ package e2e_source
...
+	kubeclient "knative.dev/pkg/client/injection/kube/client"
+
+	"knative.dev/eventing-kafka-broker/test/pkg/logging"
+	"knative.dev/reconciler-test/pkg/environment"
 )
 
+// global is the singleton instance of GlobalEnvironment. It is used to parse
+// the testing config for the test run. The config will specify the cluster
+// config as well as the parsing level and state flags.
+var global environment.GlobalEnvironment
```
Comment (Cali0707) [inline test/e2e_source/main_test.go:37]: Same as above

File: test/e2e/main_test.go [hunk 4]
```diff
@@ -22,15 +22,34 @@ package e2e
...
+	kubeclient "knative.dev/pkg/client/injection/kube/client"
+
+	"knative.dev/eventing-kafka-broker/test/pkg/logging"
+	"knative.dev/reconciler-test/pkg/environment"
 )
 
+// global is the singleton instance of GlobalEnvironment. It is used to parse
+// the testing config for the test run. The config will specify the cluster
+// config as well as the parsing level and state flags.
+var global environment.GlobalEnvironment
```
[the remark this answers is not in the document]
Comment (NeerajNagure) [reply test/e2e/main_test.go:37]: for getting a kube client we require a *testing.T type argument while calling testlib.setup().But in the main function we have got *testing.M type argument.So I am stuck here,can I please get some help?

[the remark this answers is not in the document]
Comment (Cali0707) [reply test/e2e/main_test.go:37]: Well, what we need is a context object that we can cancel after the test, as well as a kubeclient. If you look at the `testlib.ExportLogs` definition, you'll see that it gets a kubeclient with:
```go
config, err := pkgtest.Flags.GetRESTConfig()
	if err != nil {
	log.Printf("Failed to create REST config: %v\n", err)
	return
}

kubeClient, err := kubernetes.NewForConfig(config)
if err != nil {
	log.Printf("Failed to create kube client: %v\n", err)
	return
}
```
You should be able to reuse this to get a kubeclient. For the context and cancel, that is simpler as you can do something like:
```
ctx, cancel := context.WithCancel(context.Background())

**Comment by NeerajNagure** [comment]
I have made all the required changes @Cali0707

**Comment by NeerajNagure** [comment]
@Cali0707 do you want me to do any other changes?

**Review by Cali0707 — CHANGES_REQUESTED** [review]
This looks great @NeerajNagure ! One last change and then this will be good to merge:

It looks like the imports aren't correctly sorted (look at the lint errors on this PR). The easy way to fix this is to run goimports on your files (article on how to do this in VSCode here: https://hyr.mn/gofmt/, GoLand/Jetbrains here: https://www.jetbrains.com/help/go/integration-with-go-tools.html).

**Review by Cali0707 — APPROVED** [review]
/lgtm
/approve

Thanks @NeerajNagure !!

**Pull request merged**